scripts/git-hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
```

Comment:
This comment can go ♻️

Reply:
Nice one. Thanks Simon!
https://github.com/guardian/dotcom-rendering/pull/344